# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

  # Position Controllers ---------------------------------------
roll_controller:
  type: effort_controllers/JointPositionController
  joint: roll_joint
  pid: {p: 10000, i: 400, d: 1000}
pitch_controller:
  type: effort_controllers/JointPositionController
  joint: pitch_joint
  pid: {p: 10000, i: 400, d: 1000}

pankha_controller:
  type: velocity_controllers/JointVelocityController
  joint: pankha_joint
  pid: {p: 500, i: 25, d: 25}

led_port_controller:
  type: velocity_controllers/JointVelocityController
  joint: hemi1l_joint
  pid: {p: 500, i: 25, d: 25}
led_starboard_controller:
  type: velocity_controllers/JointVelocityController
  joint: hemi1r_joint
  pid: {p: 500, i: 25, d: 25}
